index.d.ts

```diff
@@ -89,7 +89,10 @@ declare namespace parser {
 declare const parser: {
     defaultOptions: parser.DefaultOptions;
     defaultDirectives: parser.Directive[];
-
+    (options?: parser.Options): (
+        content: string,
+        options?: parser.Options
```

Comment:
Looks like in this signature we cannot pass the `options` argument twice.

https://github.com/posthtml/posthtml-parser/blob/9327c7e347727cd5d1a5a95cfdbca0c01932bfd7/index.js#L128-L147

```ts
// First
('html', { options }) => Tree

// Second
({ options }) => ('html') => Tree
```